# from rest_framework.exceptions import APIException


# class PetAlreadyExistsError(APIException):
#     status_code = 409
#     default_detail = {'error': ['Pet already exists']}
